content/docs/400-guides/100-essentials/600-pipeline.mdx

```diff
@@ -60,6 +60,27 @@ In the above example, we start with an input value of `5`. The `increment` funct
 
 The result is equivalent to `subtractTen(double(increment(5)))`, but using `pipe` makes the code more readable because the operations are sequenced from left to right, rather than nesting them inside out.
 
+## flow
+
+`flow` is a "lazier" version of `pipe`. While `pipe` returns a concrete value, `flow` returns a function that returns a
+result. It allows us to reuse the same composed functions in different contexts.
```

Comment:
@nspaeth I reread the whole document again, and found that there is duplication `pipe` explanation. Therefore I added the `flow` function along with the `pipe` function, and redid the end, where everything is tied together a bit

![image](https://github.com/user-attachments/assets/9478027f-9172-470c-a0eb-2b45ace7db2e)